### ISSUE TITLE
Is the source available?

### ISSUE THREAD

**Issue by sandebert** [opto/Expression-Search-NG#115]
Hi,

I was expecting to find the source for the extension in this repo, just as the standard tends to be here on GitHub. But as far as I can tell that's not the case (except for source that is three years old, found in the releases). 

Is there a reason for this? Is the extension source not meant to be available, and the repo is only to contain the packaged extension? Perhaps the source is available on GitLab or some other place like this?

Thanks,

**Comment by hi5** [comment]
I would hope the developer makes the source(s) of the extensions available, it is perfectly understandable the dev. isn't a 100% programmer by trade and there is no reason to hide the [sausage making](https://sethrobertson.github.io/GitBestPractices/#sausage) - managing pull requests is actually quite easy if that is the worry and publishing commits using the [GitHub Desktop Client](https://desktop.github.com/) a breeze.

You don't need a perfect commit history (see sausage above). People wishing to contribute simply want to see the code so they suggest fixes and improvements (and for some piece of mind as well).  So I would encourage the dev to post the current state of the repo of the nostalgy++, expression search etc even in an "imperfect statte" (the truth is no one cares in real life)

**Comment by jbobbylopez** [comment]
Hey there, was just about to donate to this open source project, but figured I would just check out the source code first, then came here to see a 3 year old binary.  I thought maybe I had the the incorrect repository, because I know this has switched hands at least a couple of times.  It does seem like this is the latest though.

Donation page says donations go to Klaus Bücher, but it could be some of this information is out of date.   If there is a newer repo where the full sources reside, would very much love a link to that.  Cheers.

**Comment by hexadecagram** [comment]
Just before TB 115 was released, there was a popup asking for donations to keep the project alive. I contributed and now this? @opto would you care to comment?

**Comment by lstngr** [comment]
This is a duplicate of #15: .xpi files are just ZIP archives, the source is just “hidden”.
I'll take this occasion to agree that it would be more intuitive to just host the source code directly.

**Comment by opto** [comment]
yes, just use 7zip to unzip the xpi. You don't even need to rename it to zip.
Klaus
